performance/performance.md

```diff
@@ -54,6 +54,15 @@ Op-code caching can make web servers use fewer resources when running WordPress;
 
 ### Fragment Caching
 
+Fragment caching is the temporary storage of expensive server-side operations to avoid taxing web servers with repetitive tasks. WordPress doesn't fragment cache anything out-of-the-box, but it's become a common practice for potentially expensive operations such as generating Menu markup, Widget markup and slow MySQL or HTTP queries.
```

Comment:
"WordPress doesn't fragment cache anything out-of-the-box" may be too specific... For example, Transients API my qualify under some circumstances.